### ISSUE TITLE
Bluetooth proxy ?

### ISSUE THREAD

**Issue by labupps** [mikey0000/Mammotion-HA#45]
### The problem

Hi,

I do not understand how to install bluetooth proxy. I have added ESP Home as I supposed it's needed, but do not understand what is the next move. 
Do I need a physical hardware? 
Help needed, I 

### What Mammotion mower are you using?

Luba 2

### What version of the Mammotion intergration are you using?

last available, trying to install

### What version of Home Assistant Core are you running?

2024.6.4

**Comment by romeoechovictor** [comment]
Yes you need physical hardware.  I was a bit confused also at the beginning.  I bought 3 of these little devices from Amazon, flashed them and they just work. It says out of stock now, but you can look for the same elsewhere I'm sure.  Basically the Bluetooth proxy just acts like a repeater for the Bluetooth from your home assistant to the Luba.  You can add a few around various places in your home so they can "see" the Luba in any radio black spots.  They will then relay or repeat Bluetooth messages from your Luba back to home assistant.

3 pack ESP32 ESP-32S Development Board 2.4 GHz Dual Core WLAN WiFi + Bluetooth 2-In-1 Microcontroller ESP-WROOM-32 Chip CP2102 for Arduino https://amzn.eu/d/0bJlIGW1

**Comment by labupps** [comment]
Thanks for explaining, willmay be wait for Wifi. Depending on the time I find to spend on it.
